bigquery/README.md

```diff
@@ -4,3 +4,37 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+SQL queries through the BigQuery CLI:
+
+bq query '\<SQL QUERY\>'
+
+Legacy SQL dialect is set on default for the BigQuery CLI and must be disabled for the example queries to work.
+This can be done by editing ~/.bigqueryrc and adding the following lines:
+
+```
+[query]
+--use_legacy_sql=false
+```
+
+Example Queries:
+```
+# Listing overall build history
+
+SELECT * FROM `dataset.table`
+
+# Aggregating Build Counts by Status
+
+SELECT STATUS,  COUNT(*) FROM `dataset.table` GROUP BY STATUS
```

Comment:
It might be nice to have these separated into lines starting with the SQL keyword, since some lines can tend to get long if the query is huge.

To make it more readable, we can break it up like this:

SELECT STATUS, COUNT(*)
FROM `dataset.table`
GROUP BY STATUS

I'm not sure if @LOZORD has thoughts on this or if this isn't the proper convention.

Reply:
Multiline SGTM unless it's more difficult to copy and paste.